test_aicm_functions.md

```
@@ -0,0 +1,40 @@
+# AICM-BSA Usage Guide
+
+## Quick Start
+
+After the setup.sh update, restart your shell or run:
+```bash
+source ~/.bashrc
+```
+
+## Available Commands
+
+### Core Functions
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)